src/api/downloadFile.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+import { Platform, CameraRoll } from 'react-native';
+import RNFetchBlob from 'react-native-fetch-blob';
+
+import { getAuthHeader } from '../utils/url';
+import userAgent from '../utils/userAgent';
+import { Auth } from '../types';
+
+export default (url: string, auth: Auth) => {
+  if (Platform.OS === 'android') {
```

Comment:
maybe not, since that setup is for controlling the stylistic differences on both platforms; example statusBar margin, etc.